src/listen/listener-registry.js

```diff
@@ -428,104 +418,71 @@ class ListenerRegistry {
 	 * @returns {void}
 	 */
 	_triggerNextProvider( subscriptionName ) {
-		if( !this._listenInProgress[ subscriptionName ] ) {
+		const listenInProgress = this._localListenInProgress[ subscriptionName ];
+
+		if( !listenInProgress ) {
```

Comment:
Could we be more explicit here? Array truthyness in JS is muddled up quite a bit, e.g. both of these statements are true:
```
[]==false
!![]
```